src/overlays/actors/ovl_Demo_Shd/z_demo_shd_gfxdata.c

```diff
@@ -1,7 +1,7 @@
 #include "ultra64.h"
 #include "global.h"
 
-UNK_TYPE D_80991680[] = {
+static UNK_TYPE D_80991680[] = {
```

Comment:
is ovl_Demo_Shd/z_demo_shd_gfxdata.c suppose to be extracted?

Reply:
I was told we arent extract overlays on a large scale right now.